spec/index.md

```diff
@@ -1828,6 +1836,16 @@ Not including a setting in the `settings` record means not changing that field.
 
 The optional `sender_canister_version` parameter can contain the caller's canister version. If provided, its value must be equal to `ic0.canister_version`.
 
+
+### IC method `upload_chunk` {#ic-upload_chunk}
+Canisters have associated some storage space (hence forth chunk storage) where they can hold chunks of Wasm modules that are too lage to fit in a single message. This method allows the controllers of a canister to upload such chunks. The method returns the hash of the chunk that was stored. The size of each chunk should be up to 1MB. The size of the chunk store is bounded: currently it can hold up to `CHUNK_STORE_SIZE` chunks. 
```

Comment:
```suggestion
Canisters have associated some storage space (hence forth chunk storage) where they can hold chunks of Wasm modules that are too lage to fit in a single message. This method allows the controllers of a canister to upload such chunks. The method returns the hash of the chunk that was stored. The size of each chunk should be up to 1MiB. The size of the chunk store is bounded: currently it can hold up to `CHUNK_STORE_SIZE` chunks. 
```
I suppose the limit is half of a maximum message size which is 1MiB.